airbyte-integrations/connectors/source-stripe/unit_tests/resource/http/response/events.json

```diff
@@ -5,7 +5,7 @@
       "id": "evt_1OEiWvEcXtiJtvvhLaQOew6V",
       "object": "event",
       "api_version": "2020-08-27",
-      "created": 1700529213,
+      "created": 1739270181,
```

Comment:
We shouldn't have to modify this file. If the `created` value is important for a test, we should make it explicit in the test. Given that we don't want to change the behavior of the connector with this change, I assume that we shouldn't have to modify this though

Reply:
I have rolled back this change. Removed by mistake due to a message repository issue that has already been fixed.